src/assets/icons/icons.stories.mdx

```diff
@@ -0,0 +1,45 @@
+{/*
+ * Copyright 2024 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */}
+
+import { Meta, Title, IconGallery, IconItem } from '@storybook/blocks'
+
+import MiaPlatform from './MiaPlatform.svg?react'
```

Comment:
Can you configure svgr to avoid dealing with the `?react` parameter?

Reply:
Sure, done!